### PR TITLE
feature/my library header html and css

### PR DESCRIPTION
**Pull request by tojaarek**
dodany sprint icons wraz z grafikami

### CODE REVIEW

File: src/css/main.min.css
Comment (piotrlewoniec) [inline src/css/main.min.css:1]: Please, do not use external sass extensions in VSC.
Parcel has it own. Switch this off and delete folder css

File: src/css/main.min.css.map
Comment (piotrlewoniec) [inline src/css/main.min.css.map:1]: Parcel has it own. Switch this off and delete folder css

File: src/partials/my-library.html
Comment (piotrlewoniec) [inline src/partials/my-library.html:1]: 1. Please delete this line:
<link rel="stylesheet" href='../css/main.min.css' />  - it is done by parcel
2. Also please put in comment:
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/1.1.0/modern-normalize.min.css"
    integrity="sha512-wpPYUAdjBVSE4KJnH1VR1HeZfpl1ub8YT/NKx4PuQ5NmX2tKuGu6U/JRp5y+Y8XG2tV+wKQpNHVUX03MfMFn9Q=="
    crossorigin="anonymous" referrerpolicy="no-referrer" />
3. I suggest to build this file like index.html using includes for header main section and footer (footer is the same for both files, so in here we can also make include of footer file.
4. Possibly there will be problems with scripts integration - but I propose to test it  :)

File: src/sass/main.scss
Comment (piotrlewoniec) [inline src/sass/main.scss:1]: Please move this line @use './partials/header'; after line 8: @use './utils/reset';

File: src/sass/partials/_header.scss
Comment (piotrlewoniec) [inline src/sass/partials/_header.scss:1]: please consider using mixin for that:
@media (min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx)
you can find it in mediaquery also (I guess they should work :D )

**Comment by piotrlewoniec** [comment]
Please consider using awilczynski or arekwilczynski for branch naming f.e. arekwilczynski/feature/mylibrary (without white spaces)

**Review by piotrlewoniec — CHANGES_REQUESTED** [review]
Please take a look for given comments

**Review by tojaarek — COMMENTED** [review]
Changes made

**Pull request merged**